internal/pkg/service/buffer/api/service/service.go

```diff
@@ -399,28 +404,17 @@ func (s *service) Import(d dependencies.ForPublicRequest, payload *buffer.Import
 			csv = append(csv, csvValue)
 		}
 
-		// nolint:godox
-		// TODO get sliceID and fileID + use in stats.Notify
-
 		record := key.NewRecordKey(e.ProjectID, e.ReceiverID, e.ExportID, receivedAt, receivedAt)
 		err = str.CreateRecord(ctx, record, csv)
 		if err != nil {
 			errs.AppendWithPrefixf(err, `failed to create record for export "%s"`, e.ExportID)
 		}
 
-		key := key.NewSliceStatsKey(
-			e.ProjectID,
-			e.ReceiverID,
-			e.ExportID,
-			receivedAt,
-			receivedAt,
-			d.Process().UniqueID(),
-		)
```

Comment:
Slice Stats Key si implementoval ako ostatne kluce, co je logicke, ale nepotrebujeme ho:
- `NodeID`  sa pouziva sa iba pri zapise
- `NodeID` nepotrebujeme ako sucast struktury, sluzi iba na to, aby si `nodes` neprepisovali hodnoty